models/gpt2/config.py

```diff
@@ -1,4 +1,4 @@
-'''Flags and model configuration details, useful for replicating across files.'''
+"""Flags and model configuration details, useful for replicating across files."""
```

Comment:
I ran black with this source file and it replaced all single-quotes by double-quotes. I can revert if we like the old style.

Reply:
Seems reasonable